package.json

```diff
@@ -83,9 +82,8 @@
     "karma-chrome-launcher": "~3.1.0",
     "karma-coverage": "^2.2.0",
     "karma-jasmine": "~4.0.0",
-    "ng-packagr": "^9.0.0",
     "karma-jasmine-html-reporter": "^1.5.0",
-    "karma-coverage-istanbul-reporter": "~2.1.0",
+    "ng-packagr": "^12.1.1",
```

Comment:
This changes are regarding

Reply:
Before library was in angular 9 and portal application also was in 9 version but after migration library was not supporting so did these changes.